modules/networking/variables.tf

```diff
@@ -16,7 +16,7 @@ variable "network_cidr" {
 variable "network_private_subnet_cidrs" {
   type        = list(string)
   description = "(Optional) List of private subnet CIDR ranges to create in VPC."
-  default     = ["10.0.32.0/20", "10.0.48.0/20"]
+  default     = ["10.0.32.0/20", "10.0.48.0/20", "10.0.112.0/20"]
```

Comment:
🙏 Lets please not change any defaults. It should be sufficient to set the values in the example instead. 
Changing the defaults is a breaking change, since it affects current users of this module. Is this change really necessary?

Reply:
reverted